internal/app/machined/pkg/runtime/v1alpha1/platform/equinixmetal/equinix.go

```diff
@@ -110,9 +110,18 @@ func (p *EquinixMetal) Mode() runtime.Mode {
 }
 
 // KernelArgs implements the runtime.Platform interface.
-func (p *EquinixMetal) KernelArgs() procfs.Parameters {
-	return []*procfs.Parameter{
-		procfs.NewParameter("console").Append("ttyS1,115200n8"),
+func (p *EquinixMetal) KernelArgs(arch string) procfs.Parameters {
+	switch arch {
+	case "amd64":
+		return []*procfs.Parameter{
+			procfs.NewParameter("console").Append("ttyS1,115200n8"),
+		}
+	case "arm64":
+		return []*procfs.Parameter{
+			procfs.NewParameter("console").Append("ttyAMA0,115200"),
```

Comment:
this is documented here: https://deploy.equinix.com/developers/docs/metal/operating-systems/custom-ipxe/